src/pages/guides/getting_started/changelog.md

```diff
@@ -22,6 +22,26 @@ contributors:
 
 # Changelog
 
+## 2025-11-24
+
+### Code Playground Updates
+
+The Adobe Express Code Playground has received a major update with the release of a new set of features and improvements, which this site now reflects.
+
+- The Code Playground now supports [Cloud-based session persistence](./code-playground-workflow.md#session-management) (instead of the previous local storage) allowing you to save your work and resume where you left off everywhere. The new **More** menu contains items to manage your sessions (rename or delete), or star new ones.
+- Relevant Code Blocks (e.g. in the [How-to guides](../learn/how_to/use_text.md)) now feature a **Try in playground** button that opens a new Playground Script session with the code pre-filled for easier testing.
```

Comment:
Can we rephrase this a bit? Something along the lines of 'Now you can try out the code blocks (e.g. in the [How-to guides](../learn/how_to/use_text.md)) by simply clicking on the **Try in playground** button